TEST_SETUP_GUIDE.md

```
@@ -0,0 +1,262 @@
+### 📦 MinIO Configuration
+```bash
+# MinIO is now the default storage provider
+STORAGE_PROVIDER=minio
+MINIO_ENDPOINT=http://localhost:9000
+MINIO_ACCESS_KEY=minioadmin
+MINIO_SECRET_KEY=minioadmin
+MINIO_BUCKET=mybudget-uploads
+```
+
+### 🚀 Storage Service Integration
+- **Console Access**: http://localhost:9001 (admin/admin)
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)